modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -3816,6 +3849,9 @@ private void _testUpdateObjectEntryObjectStateTransitions()
 	@Inject
 	private DLFileEntryLocalService _dlFileEntryLocalService;
 
+	@DeleteAfterTestRun
+	private ObjectDefinition _draftObjectDefinition;
```

Comment:
Hi @thallescezar , maybe we should delete this on `tearDown` instead of rely on `DeleteAfterTestRun` (https://github.com/liferay-objects/liferay-portal/blob/2b26e6c26c7521cd250762fb5b296409734ef760/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L420)